File: swagger/models/direct_debit_info.py
from .base_model import BaseModel


class DirectDebitInfo(BaseModel):
	"""
	no description available

	Attributes:

		name_on_account
		The name on the bank account.

		routing_number
		The routing number for the bank.

		account_number
		The bank account number.

		account_type
		The account type. Either Checking or Savings.
	"""

	input_map = {
		'NameOnAccount': 'name_on_account',
		'RoutingNumber': 'routing_number',
		'AccountNumber': 'account_number',
		'AccountType': 'account_type',
		}

	output_map = {
		'name_on_account': 'NameOnAccount',
		'routing_number': 'RoutingNumber',
		'account_number': 'AccountNumber',
		'account_type': 'AccountType',
		}

File: swagger/Models/DirectDebitInfo.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class DirectDebitInfo extends BaseModel
{
	/**
	 * The name on the bank account.
	 * 
	 * @var string 
	 */
	public $nameOnAccount;
	/**
	 * The routing number for the bank.
	 * 
	 * @var string 
	 */
	public $routingNumber;
	/**
	 * The bank account number.
	 * 
	 * @var string 
	 */
	public $accountNumber;
	/**
	 * The account type. Either Checking or Savings.
	 * 
	 * @var string 
	 */
	public $accountType;

	protected function getInputMap()
	{
		return [
			'NameOnAccount' => ['nameOnAccount', 'string', null],
			'RoutingNumber' => ['routingNumber', 'string', null],
			'AccountNumber' => ['accountNumber', 'string', null],
			'AccountType' => ['accountType', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'nameOnAccount' => ['NameOnAccount', 'string', null],
			'routingNumber' => ['RoutingNumber', 'string', null],
			'accountNumber' => ['AccountNumber', 'string', null],
			'accountType' => ['AccountType', 'string', null],
			];
	}
}
